File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1165.cpp
#include <iostream>

using namespace std;

int main(){
    int n, v, i, j;
    cin >> n;
    for(j=0;j<n;j++){
        cin >> v;
        int div=0;
        if(v==2){
            cout << v << " eh primo" << endl;
        } else if(v==1){
            cout << v << " nao eh primo" << endl;
        } else if(v%2==0){
            cout << v << " nao eh primo" << endl;
        } else{
            for(i=3;i<v;i+=2){
                if(v%i==0){
                    div=1;
                    break;
                }
            }
            if(div==0){
                cout << v << " eh primo" << endl;
            } else{
                cout << v << " nao eh primo" << endl;
            }
        }
    }

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1165.rb
n = gets.to_i
for j in 1..n do
    v = gets.to_i
    div=0
    if(v==2) then
        puts("#{v} eh primo")
    elsif(v==1) then
        puts("#{v} nao eh primo")
    elsif(v%2==0) then
        puts("#{v} nao eh primo")
    else
        for i in 3..(v-1) do
            if(v%i==0) then
                div=1
                break
            end
            i+=1
        end
        if(div==0) then
            puts("#{v} eh primo")
        else
            puts("#{v} nao eh primo")
        end
    end
end